lib/_http_client.js

```diff
@@ -24,19 +24,21 @@
 const util = require('util');
 const net = require('net');
 const url = require('url');
-const HTTPParser = process.binding('http_parser').HTTPParser;
+const { HTTPParser } = process.binding('http_parser');
 const assert = require('assert').ok;
 const common = require('_http_common');
-const httpSocketSetup = common.httpSocketSetup;
-const parsers = common.parsers;
-const freeParser = common.freeParser;
-const debug = common.debug;
-const OutgoingMessage = require('_http_outgoing').OutgoingMessage;
+const {
+  httpSocketSetup,
+  parsers,
+  freeParser,
+  debug
+} = common;
```

Comment:
I suppose you can add `_checkIsHttpToken: checkIsHttpToken` too here, which would get rid of the only reference to `common` in the file other than this one. Also would you mind sorting these in alphabetical order?